packages/cli/src/handlers/ibc/index.ts

```
@@ -0,0 +1,89 @@
+        },
+      },
+      {
+        requestMessage: "Input channel-id:",
+        validate: (input: string) => {
+          return input.startsWith("channel-");
+        },
+      },
+      {
+        requestMessage: "Input packet-sequence:",
+        validate: (input: string) => {
+          return !isNaN(parseInt(input));
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)